supermercado/scripts/cli.py

```diff
@@ -42,22 +45,28 @@ def union(inputtiles, parsenames):
     for u in unioned:
         click.echo(json.dumps(u))
 
-cli.add_command(union)
-
 
-@click.command('burn')
+@cli.command('burn')
 @cligj.features_in_arg
 @cligj.sequence_opt
-@click.argument('zoom', type=int)
+@click.argument(
+    'zoom',
+    type=str
```

Comment:
The downside of this is that we lose click's argument validation -- we'd need to add a bunch of checks for:
- is the string formatted correctly
- is the parsed output integers
- are they in the right order (eg what if someone wrote `10..0`)

Reply:
Thanks @dnomadb, 
I could create a custom class for that like https://github.com/mapbox/rio-glui/blob/master/rio_glui/scripts/cli.py#L35-L52

Reply:
> are they in the right order (eg what if someone wrote 10..0)

well this might be tricky but we can use `min()` and `max()` to make sure of the order